backend/infrahub/message_bus/operations/requests/proposed_change.py

```diff
@@ -134,6 +134,12 @@ async def pipeline(message: messages.RequestProposedChangePipeline, service: Inf
 
         await _gather_repository_repository_diffs(repositories=repositories)
 
+        destination_branch = await registry.get_branch(db=service.database, branch=message.destination_branch)
+        source_branch = await registry.get_branch(db=service.database, branch=message.source_branch)
+        component_registry = get_component_registry()
+        async with service.database.start_transaction() as dbt:
+            diff_coordinator = await component_registry.get_component(DiffCoordinator, db=dbt, branch=source_branch)
+            await diff_coordinator.update_branch_diff(base_branch=destination_branch, diff_branch=source_branch)
```

Comment:
This would only update the branch diff and not in any way give us access to the diff_summary right? I'm thinking that we first do some operations directly against the database and then as the next step we use the SDK to query for the changes (which I guess would be what we updated but also what might have been updated earlier)

Reply:
the `DiffTree` and `DiffTreeSummary` graphql queries get their data from the same place. Running `update_branch_diff` here will create/update that data so that both queries can be used and will be up-to-date (for the moment, at least. other stuff that runs in the pipeline can make changes that will require another update to the diff